libmozdata/vcs_map.py

```diff
@@ -14,10 +17,16 @@
 
 def download_mapfile():
     if not os.path.exists(VCS_MAP_FULL_PATH):
-        urllib.request.urlretrieve(
-            f"{MAPPER_SERVICE}/gecko-dev/mapfile/full", VCS_MAP_FULL_PATH
+        file_name, _ = urllib.request.urlretrieve(
+            "https://moz-vcssync.s3-us-west-2.amazonaws.com/mapping/gecko-dev/git-mapfile.tar.bz2"
         )
 
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            with tarfile.open(file_name, "r:bz2") as tar:
+                tar_path = "./build/conversion/beagle/.hg/git-mapfile"
+                tar.extract(member=tar_path, path=tmp_dir)
```

Comment:
Use tar.extractfile and then write the buffer in VCS_MAP_FULL_PATH would be better